src/Apps/Order2/Routes/Checkout/Components/Order2PaymentStep.tsx

```diff
@@ -0,0 +1,125 @@
+import CheckmarkIcon from "@artsy/icons/CheckmarkIcon"
+import { Clickable, Flex, Spacer, Text } from "@artsy/palette"
+import { Order2PaymentForm } from "Apps/Order2/Routes/Checkout/Components/Order2PaymentForm"
+import { Brand, BrandCreditCardIcon } from "Components/BrandCreditCardIcon"
+import type { Order2PaymentForm_order$key } from "__generated__/Order2PaymentForm_order.graphql"
+import { useState } from "react"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { graphql } from "react-relay"
+import { Order2PaymentStepConfirmationTokenQuery } from "__generated__/Order2PaymentStepConfirmationTokenQuery.graphql"
+
+interface Order2PaymentStepProps {
+  order: Order2PaymentForm_order$key
+}
+
+export const Order2PaymentStep: React.FC<Order2PaymentStepProps> = ({
```

Comment:
Let's rebase and then organize this file similar to how the FulfillmentDetailsStep is in a single directory.

Reply:
In the process I think `MeConfirmationTokenQueryRenderer` can be renamed to something about the step being complete.